interactive-clap-derive/src/derives/interactive_clap/mod.rs

```diff
@@ -337,9 +371,7 @@ pub fn impl_interactive_clap(ast: &syn::DeriveInput) -> TokenStream {
                         <#cli_name as clap::Parser>::parse()
                     }
 
-                    pub fn try_parse_from(s: &str) -> Result<#cli_name, clap::Error> {
-                        <#cli_name as clap::Parser>::try_parse_from(s.split(" "))
-                    }
```

Comment:
Oh my, this method should take an iterator like [`try_parse_from` in clap](https://docs.rs/clap/latest/clap/trait.Parser.html#method.try_parse_from), and just pass it through like `parse` and `try_parse` do, not `&str`. Let's make it a breaking API change and release it under a new version instead of having the feature flag. Splitting the arguments should happen outside of interactive-clap.

Reply:
Done!